SWADroid/src/main/res/values/lists.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
 
+    <string-array name="gameMenuItems">
+        <item>Active games</item>
+    </string-array>
```

Comment:
A menu with only one item does not make sense. Maybe you can simplify the design of the UI to eliminate it.